stubs/WTForms/wtforms/fields/datetime.pyi

```diff
@@ -0,0 +1,116 @@
+        name: str | None = None,
+        _form: BaseForm | None = None,
+        _prefix: str = "",
+        _translations: _SupportsGettextAndNgettext | None = None,
+        _meta: DefaultMeta | None = None,
+    ) -> None: ...
+
+class DateField(DateTimeField):
+    data: date | None  # type: ignore[assignment]
+    default: date | Callable[[], date] | None  # type: ignore[assignment]
+    def __init__(
+        self,
+        label: str | None = None,
+        validators: tuple[_Validator[_FormT, Self], ...] | list[Any] | None = None,
+        format: str | list[str] = "%Y-%m-%d",
+        *,
+        filters: Sequence[_Filter] = (),
+        description: str = "",
+        id: str | None = None,
+        default: date | Callable[[], date] | None = None,
+        widget: _Widget[Self] | None = None,
+        render_kw: dict[str, Any] | None = None,
+        name: str | None = None,
+        _form: BaseForm | None = None,
+        _prefix: str = "",
+        _translations: _SupportsGettextAndNgettext | None = None,
+        _meta: DefaultMeta | None = None,
+    ) -> None: ...
+
+class TimeField(DateTimeField):
+    data: time | None  # type: ignore[assignment]
+    default: time | Callable[[], time] | None  # type: ignore[assignment]
+    def __init__(
+        self,
+        label: str | None = None,
+        validators: tuple[_Validator[_FormT, Self], ...] | list[Any] | None = None,
+        format: str | list[str] = "%H:%M",
+        *,
+        filters: Sequence[_Filter] = (),
+        description: str = "",
+        id: str | None = None,
+        default: time | Callable[[], time] | None = None,
+        widget: _Widget[Self] | None = None,
+        render_kw: dict[str, Any] | None = None,
+        name: str | None = None,
+        _form: BaseForm | None = None,
+        _prefix: str = "",
+        _translations: _SupportsGettextAndNgettext | None = None,
+        _meta: DefaultMeta | None = None,
+    ) -> None: ...
+
+class MonthField(DateField):
+    def __init__(
+        self,
+        label: str | None = None,
+        validators: tuple[_Validator[_FormT, Self], ...] | list[Any] | None = None,
+        format: str | list[str] = "%Y-%m",
+        *,
+        filters: Sequence[_Filter] = (),
+        description: str = "",
+        id: str | None = None,
+        default: time | Callable[[], time] | None = None,
+        widget: _Widget[Self] | None = None,
+        render_kw: dict[str, Any] | None = None,
+        name: str | None = None,
+        _form: BaseForm | None = None,
+        _prefix: str = "",
+        _translations: _SupportsGettextAndNgettext | None = None,
+        _meta: DefaultMeta | None = None,
+    ) -> None: ...
+
+class DateTimeLocalField(DateTimeField):
+    def __init__(
+        self,
+        label: str | None = None,
+        validators: tuple[_Validator[_FormT, Self], ...] | list[Any] | None = None,
+        format: str | list[str] = ...,
+        *,
+        filters: Sequence[_Filter] = (),
```

Comment:
This would be a really nice use case for PEP 692

Reply:
Indeed, `Unpack` should not be used yet in typeshed though, or has this recently changed?

It may be worth to leave a comment though.

Reply:
Can't be used yet, no, see #9710